services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.base_user import AbstractBaseUser
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> AbstractBaseUser:
```

Comment:
The `get_user` function should return an instance of the custom `User` model instead of `AbstractBaseUser`. Consider using `get_user_model()` to ensure compatibility with custom user models.